libs/core-components/src/lib/Form/FormField.jsx

```diff
@@ -0,0 +1,170 @@
+import React, { useState } from 'react';
+import {
+  FormGroup,
+  Label,
+  Input,
+  FormText,
+  Badge,
+  InputGroup,
+} from 'reactstrap';
+import Button from '../Button';
+
+import { useField } from 'formik';
+import PropTypes from 'prop-types';
+import './FormField.scss';
```

Comment:
Requests:

1. Rename to `FormField.global.css`. [Reference.](https://github.com/TACC/tup-ui/blob/task/tup-ui-styles/apps/tup-ui/src/styles/README.md)
2. Convert SCSS syntax—if any—to native CSS or  [PostCSS syntax](https://github.com/csstools/postcss-plugins/tree/main/plugin-packs/postcss-preset-env#glossary).